### ISSUE TITLE
Evaluation error when dataset has slash in name

### ISSUE THREAD

**Issue by byquip** [MPh-py/MPh#66]
Hello, I was trying to retrieve some data from my .mph project but got error which I have no idea how to handle.
Is there way to fix it from my side?

```
import mph
client = mph.start(cores=1)
model = client.load("my_path")
sols = model.solutions()
dats = model.datasets()
print(sols)
print(dats)
for dat in dats:
    print(dat)
    s = model.outer(dat)
    print(s)
```
OUTPUT:
```
Traceback (most recent call last):
  File "C:/Users/n/Desktop/d/comsoltest.py", line 10, in <module>
    s = model.outer(dat)
  File "C:\Users\n\AppData\Local\Programs\Python\Python38\lib\site-packages\mph\model.py", line 407, in outer
    if not dataset.exists():
  File "C:\Users\n\AppData\Local\Programs\Python\Python38\lib\site-packages\mph\node.py", line 278, in exists
    return (self.java is not None)
  File "C:\Users\n\AppData\Local\Programs\Python\Python38\lib\site-packages\mph\node.py", line 212, in java
    return eval(self.groups.get(name))
TypeError: eval() arg 1 must be a string, bytes or code object
['Solution 1', 'Parametric Solutions 1', 'wl=1.15E-6']
['Study 1//Solution 1', 'Study 1//Parametric Solutions 1']
Study 1//Solution 1

Process finished with exit code 1
```

**Comment by byquip** [comment]
removing slash "//" solved this problem.

**Comment by john-hen** [comment]
You know, this may actually be a bug. But I've only briefly looked at your traceback and you've already closed the issue. I may give this a second look some other time.

**Comment by wakass** [comment]
Having the same issue, removing the slash from the dataset name inside COMSOL is a workaround. Would be great if this could be fixed. Slashes seem to be standard when parametrizing a study. 

Windows 10, COMSOL 5.6 btw.

**Comment by john-hen** [comment]
Fixed in version 1.1.2, released today.
